packages/alea-frontend/pages/api/get-durations.ts

```diff
@@ -0,0 +1,47 @@
+import fs from 'fs/promises';
```

Comment:
please rename this to be more descriptive. eg. "get-teaching-duration-per-section" or